src/main/java/com/denizenscript/depenizen/bukkit/objects/breweryx/BPlayerTag.java

```diff
@@ -0,0 +1,161 @@
+package com.denizenscript.depenizen.bukkit.objects.breweryx;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.dre.brewery.BPlayer;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+// TODO: OfflinePlayer support
+public class BPlayerTag implements ObjectTag {
```

Comment:
This whole object doesn't make sense -- this should just be an extension of PlayerTag

also identifying players by name hasn't been permissible in Minecraft in around 10 years now, since Minecraft 1.7.6 added UUIDs